### PR TITLE
Adding HEDA to projects, NPSP URL

### PR DESCRIPTION
**Pull request by dylanhansen**
Getting my environment on my desktop setup, using minor project updates to test.

@dextermilo Let me know if you have any questions or if you'd like to be involved in review of these minor changes.  Thanks!

### CODE REVIEW

**Comment by dextermilo** [comment]
@dylanhansen any issues getting it going?

File: _data/projects.yml [hunk 1]
```diff
@@ -1,6 +1,12 @@
 - name:           NPSP
   description:        The next version of the Salesforce.org Nonprofit Starter Pack.
```
Comment (dextermilo) [inline _data/projects.yml]: update this?

Comment (dylanhansen) [inline _data/projects.yml]: Updated - tried to keep it brief so it looks good on the page.

**Comment by dylanhansen** [comment]
@dextermilo No issues getting Jekyll running locally to test changes.  Curious if we should add the Gemfiles that get created to .gitignore?

**Pull request merged**